python/src/opendp/_lib.py

```diff
@@ -26,16 +26,21 @@ def _load_library():
         lib_dir = Path(__file__).parent / ".." / ".." / ".." / 'rust' / 'target' / build_dir  # pragma: no cover
 
     if lib_dir.exists():
+        from importlib.machinery import EXTENSION_SUFFIXES
+        suffixes = EXTENSION_SUFFIXES + [".dylib"]
+
         lib_dir_file_names = [
```

Comment:
```suggestion
        lib_paths = [
```
Since these are no longer just names: They are complete paths.